README.md

```diff
@@ -16,4 +16,36 @@ the *simpleHTMLChangeLogFilename* parameter to *gitlog.html*.
 Usage instructions and Documentation
 ====================================
 
-See the **[Maven Gitlog Plugin documentation](http://danielflower.github.io/maven-gitlog-plugin/)** for usage and more information.
\ No newline at end of file
+See the **[Maven Gitlog Plugin documentation](http://danielflower.github.io/maven-gitlog-plugin/)** for usage and more information.
+
+
+Merge Log Only

```

Comment:
Can you add this section and the AsciiDoc section to `src/site/markdown/usage.md.vm` so that the site docs will be updated automatically on release?